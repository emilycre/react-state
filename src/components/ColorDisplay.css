.ColorDisplay {
  width: 5em;
  height: 5em;
  margin: 1em;
  border-radius: 50%;
}
