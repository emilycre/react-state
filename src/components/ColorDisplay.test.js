import React from 'react';
import { shallow } from 'enzyme';
import ColorDisplay from './ColorDisplay';

describe('ColorDisplay component', () => {
  it('Renders a ColorDisplay', () => {
    const wrapper = shallow(<ColorDisplay color="#000000" />);
    expect(wrapper).toMatchSnapshot();
  });
});
