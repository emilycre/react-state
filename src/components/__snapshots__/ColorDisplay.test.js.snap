// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorDisplay component Renders a ColorDisplay 1`] = `
<div
  className="ColorDisplay"
  style={
    Object {
      "backgroundColor": "#000000",
    }
  }
/>
`;
